example/src/main/AndroidManifest.xml

```diff
@@ -2,11 +2,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.klarna.example" >
 
+    <uses-permission android:name="android.permission.INTERNET" />
+
     <application
         android:allowBackup="true"
         android:icon="@drawable/ic_launcher"
-        android:label="@string/app_name"
-        android:theme="@style/AppTheme" >
+        android:theme="@android:style/Theme.Holo.Light">
```

Comment:
in the near future I suggest you should evolve to Material Design :) 
api 21 and above - will be more appealing probably.